R/json-joins.R

```diff
@@ -0,0 +1,40 @@
+#' JSON joins
+#'
+#' These are wrappers around `pack_join()` and `dplyr::nest_join()` which store
+#' the joined data into a json column.
+#' `json_pack_join()` returns all rows and columns in x with a new json columns that contains all packed matches from y.
+#' `json_nest_join()` returns all rows and columns in x with a new json columns that contains all nested matches from y.
+#'
+#' @inheritParams dplyr::nest_join
+#' @export
+json_nest_join <- function(x, y, by = NULL, ..., copy = FALSE, keep = FALSE, name = NULL) {
```

Comment:
Can you please add examples?

Reply:
done in https://github.com/cynkra/dm/pull/918/commits/4f51a4098013f885f8a619e05f5fd00673828c7a